scout/server/blueprints/variants/templates/variants/utils.html

```diff
@@ -49,16 +55,22 @@
 
 {% macro filter_form_footer(form, result_size, total_variants, more_variants, page, nvars) %}
 <div class="card-footer text-center">
+  <input type="hidden" id="expand_search" name="expand_search">
   <div class="row">
-    <div class="col-2 d-flex align-items-baseline justify-content-start">
-      <input type="hidden" id="expand_search" name="expand_search">
-      {{ form.hide_dismissed }}
-      {{ form.hide_dismissed.label(class="ml-3") }}
+    <div class="col-4">
+      <div class="form-check d-flex justify-content-start">
+        {{ form.hide_dismissed(class="form-check-input") }}
+        {{ form.hide_dismissed.label(class="form-check-label") }}
+      </div>
+      <div class="form-check d-flex justify-content-start">
+        {{ form.show_unaffected(class="form-check-input") }}
+        {{ form.show_unaffected.label(class="form-check-label", **{'data-toggle': 'tooltip', 'title':'This filter applies only to cases with more than one sample'}) }}
```

Comment:
Perhaps explain also the general function? E.g. 
```suggestion
        {{ form.show_unaffected.label(class="form-check-label", **{'data-toggle': 'tooltip', 'title':'This filter applies only to cases with more than one sample. When this is unchecked, only variants that are predicted by callers to be present in an affected individual genotype are show.'}) }}
```